broker/src/main/java/io/camunda/zeebe/broker/bootstrap/JobStreamServiceStep.java

```diff
@@ -115,7 +116,6 @@ public String getName() {
     return "JobStreamService";
   }
 
-  // TODO: replace with real activation properties
   private record DummyActivationProperties(
```

Comment:
:question: Why don't we completely remove the `DummyActivationProperties` class and replace it with `JobActivationPropertiesImpl` on line 49, when the `RemoteStreamService` is created?